src/plugins/companionTranslations.test.ts

```diff
@@ -2,15 +2,20 @@ import path from 'path';
 
 import companionTranslations from './companionTranslations';
 
-const basePath = path.join(__dirname, '__test__', 'companionTranslations');
+const normalizePath = (val: string) => val.replace(/[/\\]/g, '/');
```

Comment:
This is duplicated in another file, factor it out into another shared file.

Reply:
Something like *src/utils/test.ts*?